lib/tasks/facts.rake

```diff
@@ -6,12 +6,13 @@ This is useful when you used custom facts of different facts sourcers that you d
 After cleaning of such facts they should no longer appear in auto-completion suggestions.
 
 Examples:
-  # foreman-rake facts:clean
+  # foreman-rake facts:clean [batch_size=500]
 END_DESC
 
   task :clean => :environment do
-    puts 'Starting orphaned facts clean up...'
-    count = FactCleaner.new.clean!.deleted_count
+    batch_size = ENV['batch_size'] ? ENV['batch_size'].to_i : 500
```

Comment:
I think it would be a bit cleaner to use official rake [parameters API](https://ruby.github.io/rake/doc/rakefile_rdoc.html#label-Tasks+that+Expect+Parameters):

``` ruby
task :clean[:batch_size] => :environment do
```

Reply:
It does not work on production installs for some reason.

Reply:
From reading more threads about it, I see it's really sensitive to spaces (and for zsh, you have to escape `[]` too).

Let's leave it as `ENV`.

I have considered `OptionsParser`, but I think it would be over-complication for this use case.